packages/cli/src/bin.ts

```diff
@@ -5,7 +5,7 @@ export const run = async () => {
   const cli = build()
     .brand("webstone")
     .src(__dirname)
-    .plugins("./node_modules", { matching: "webstone-plugin-cli-*" })
+    .plugins("./node_modules", { matching: "webstone-plugin-*-cli" })
```

Comment:
This change makes #286 redundant I think. We can close the PR in favor of this one